### PR TITLE
feat: Add dataset language

### LINKED ISSUE #1105: Document language of sources
Each source is configured with as static language and the label are returned in this language but the language of labels is not included in GraphQL results. Please add a catalog field `language` for each source and configure the language in `catalog/datasets/`, e.g.

~~~json
"language": { "@value": "nl" }
~~~

So query

~~~graphql
query {
  sources {
    uri
    name
    description
    language
  }
}
~~~

results in

~~~json
    {
        "uri": "https://druid.datalegend.net/AdamNet/Geography/sparql#streets",
        "name": "Adamlink: straten in Amsterdam",
        "description": "Straten in Amsterdam",
        "language": "nl"
    }
~~~

### PR DESCRIPTION
**Pull request by ddeboer**
* Add required property `schema:inLanguage` to datasets.
* Update catalog.
* Expose language as a list for future extensibility.

Fix #1105

### CODE REVIEW

**Comment by ddeboer** [comment]
@EnnoMeijers Please review whether the catalog changes are correct for each dataset.

File: packages/network-of-terms-graphql/src/schema.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export const schema = `
     description: String!
     creators: [Creator]!
     features: [Feature]!
+    inLanguage: [String]!
```
Comment (ddeboer) [inline packages/network-of-terms-graphql/src/schema.ts:12]: Made this a list so we can add multiple language support in the future without breaking backwards compatibility in the GraphQL API.

File: packages/network-of-terms-catalog/catalog/datasets/aat-materials.jsonld [hunk 2]
```diff
@@ -28,6 +28,10 @@
       "@value": "Selectie van AAT termen voor het beschrijven van materialen in architectuur-, kunst- en cultuurhistorische collecties"
     }
   ],
+  "inLanguage": {
+    "@type": "Language",
+    "alternateName": "nl"
+  },
```
[the remark this answers is not in the document]
Comment (coret) [reply packages/network-of-terms-catalog/catalog/datasets/aat-materials.jsonld:34]: The expected type of [inLanguage](https://schema.org/inLanguage) is Language or Text. You're proposing the Language form for the Catalog. In the [Dataset requirements](https://netwerk-digitaal-erfgoed.github.io/requirements-datasets/#dataset-example) example we use the Text form (but do include a link to [IETF BCP 47 standard](http://tools.ietf.org/html/bcp47). If we're consistent we should update the  Dataset Register example and maybe add a check to the SHACL for this property (warning, not error)?

**Review by EnnoMeijers — COMMENTED** [review]
Geonames is set to "en" but is kind of mixed, it returns both Dutch and English label such as "Gemeente Den Haag" and "The Hague", not a big deal for now. 

Iconclass is set to "nl" but returns English labels, so this should be "en".

**Pull request merged**